rollup.release.config.js

```diff
@@ -16,17 +16,7 @@ const config = buildRollupConfig(baseRollupConfig => {
       debugger: true,
       functions: [
         'console.*',
-        'Logger.trace',
-        'Logger.dir',
-        'Logger.time',
-        'Logger.timeEnd',
-        'Logger.group',
-        'Logger.groupEnd',
-        'Logger.assert',
-        'Logger.log',
-        'Logger.info',
-        'Logger.warn',
-        'Logger.deprecate'
```

Comment:
Would removing this here remove for all bundles? Can this be conditionally done somehow?

Reply:
I think this should be for all bundles
@lesterchoi-okta What's your opinion?

Reply:
@glenfannin-okta what would the condition be?

Reply:
Actually, after looking through it, I think this is sufficient: https://github.com/okta/okta-signin-widget/pull/3546/files#diff-bf3aa7d3e0f122607ccc9a328eda2b2409fbb00d66959e285e0a5aa222cd5909R17

Basically, I just wanted to be sure we arent exposting these commands when we didnt intend to in production environments. I think the above changes achieves that.